src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -191,6 +191,11 @@ function validate_options_type_by_value(input_options_with_data) {
             const details = `type of flag ${option} should be ${type_of_option} (and the received value is ${value})`;
             throw_cli_error(ManageCLIError.InvalidArgumentType, details);
         }
+        // ensure access_keys values are not empty or whitespace only strings
+        if (['access_key', 'secret_key'].includes(option) && value.trim() === '') {
```

Comment:
this function tests the type of the variables. I am not sure if its the best place to put this check. maybe move it to validate_account_identifier. @shirady WDYT

Reply:
also does is this case only a problem for access_key, secret_key, or for all string options? if not maybe its better to have a more general check

Reply:
or we can put it to `validate_account_args` , I found it more suitable for this case.

Currently we are facing this only for access_key and secret_key as this values cannot be unset with ''(empty string). @shirady pls correct me if I am wrong.

Reply:
I agree with @nadavMiz comment about moving the implementation from the function `validate_options_type_by_value`.

@achouhan09 from what I see in the issue and know those are the cases, but maybe there are more.
You can move the core implementation to a function, and if we will have more cases we can reuse it.

Reply:
Done, Thanks